demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -291,3 +298,57 @@ def is_using_instance(self) -> bool:
                     self.is_valid = False
                     return False
         return True
+
+    def is_script_id_valid(self, skip_dependencies=False):
+        """Checks whether a script id is valid (i.e id exists in set_id)
+        Args:
+            skip_dependencies (bool):  whether should skip id set validation or not
+        Return:
+            bool. if all scripts ids of this playbook are valid.
+        """
+        is_valid = True
+        id_set = self.get_id_set_file(skip_dependencies)
+        if not id_set:
```

Comment:
Shouldn't this case cause an exception?
When would this case happen? When does ID set creation fail?